src/Core/Utils/Color.hpp

```diff
@@ -10,12 +10,20 @@ namespace Ra {
 namespace Core {
 namespace Utils {
 
+/**
+ * Get a named color from its name. Return the uint value of named svg color. The value is of the
+ * form 0xRRGGBB. RR is red, GG is green and BB is blue. \param name: the name of the color as given
+ * from svg spec. \return Color code, or 0 if \a name is not a valid svg color name (0 is
+ * corresponds to black). \see https://www.december.com/html/spec/colorsvg.html
```

Comment:
```suggestion
 * Get a named color from its name. Return the uint value of named svg color. The value is of the
 * form 0xRRGGBB. RR is red, GG is green and BB is blue. 
 * 
 * \param name: the name of the color as given from svg spec. 
 * \return Color code, or 0 if \a name is not a valid svg color name (0 corresponds to black). 
 * \see https://www.december.com/html/spec/colorsvg.html
```

Reply:
maybe the clang-format will reformat this (at least when no newline... but let's try.